packages/model-tuning/src/model_tuning/embedding.py

```diff
@@ -37,6 +60,27 @@ def embed_loinc_names(
     # Make sure just the directory exists
     os.makedirs(EMBEDDING_CACHE_DIR, exist_ok=True)
 
+    model_dir = os.path.join(EMBEDDING_CACHE_DIR, dest)
+    os.makedirs(model_dir, exist_ok=True)
+
+    current_chunk = []
+    chunk_index = 0
+
+    def flush_chunk():
+        nonlocal current_chunk, chunk_index
+        if not current_chunk:
+            return
+        chunk_index += 1
+        chunk_filename = os.path.join(
+            model_dir,
+            f"{dest}_embeddings_chunk_{chunk_index:04d}.jsonl",
+        )
+        with open(chunk_filename, "w") as f:
+            for doc in current_chunk:
+                f.write(json.dumps(doc) + "\n")
```

Comment:
This is pretty inconsequential so feel free to ignore, but you could simplify these two lines.
```suggestion
            f.writelines(json.dumps(doc) + "\n" for doc in current_chunk)
```